src/sagemaker/jumpstart/notebook_utils.py

```diff
@@ -141,6 +139,20 @@ def extract_framework_task_model(model_id: str) -> Tuple[str, str, str]:
     return framework, task, name
 
 
+def extract_model_type(spec_key: str) -> str:
```

Comment:
shouldn't this return a `JumpStartModelType` ?

Reply:
Updated function naming, this returns the filter representation of model type, to allow `list_jumpstart_models("model_type is proprietary")`